analyzer.go

```diff
@@ -42,6 +45,7 @@ func (a Analyzer) DeriveManifestAndLock(path string, n gps.ProjectRoot) (gps.Man
 	return m, nil, nil
 }
 
+// Info returns the name and version of this ProjectAnalyzer.
```

Comment:
...god what a terrible method name, i thought i'd gotten annoyed at this and improved it before 😢 will just have to do it later